platforms/android/library/src/main/java/io/element/android/wysiwyg/spans/CodeSpanConstants.kt

```diff
@@ -0,0 +1,5 @@
+package io.element.android.wysiwyg.spans
+
+internal object CodeSpanConstants {
+    const val DEFAULT_RELATIVE_SIZE_PROPORTION = 0.87f
```

Comment:
0.87 is the default size for code in Element Android (the value comes [from Markwon](https://github.com/noties/Markwon/blob/2ea148c30a07f91ffa37c0aa36af1cf2670441af/markwon-core/src/main/java/io/noties/markwon/core/MarkwonTheme.java#L106))